lib/etc/openturns.conf.in

```diff
@@ -462,8 +462,8 @@
   <MaternModel-DefaultNu    value="1.5"   />
   <MaternModel-DefaultTheta value="10.0"  />
   
-  <!-- OT::WeightedExperiment parameters -->
-  <WeightedExperiment-DefaultSize value="100" />
+  <!-- OT::WeightedExperimentImplementation parameters -->
+  <WeightedExperimentImplementation-DefaultSize value="100" />
```

Comment:
IMHO we should remove `Implementation` substrings from all resource keys (not in this PR, this is a general comment).  It does not add any value, and is annoying when performing changes like this one.